runtime/doc/lua.txt

```diff
@@ -2392,4 +2392,35 @@ trust({opts})                                             *vim.secure.trust()*
         • true and full path of target file if operation was successful
         • false and error message on failure
 
+
+==============================================================================
+Lua module: inspector                                          *lua-inspector*
+
+get({bufnr}, {row}, {col}, {filter})                     *vim.inspector.get()*
+    Get all the items at a given buffer position
```

Comment:
Since this is a "get all the things" interface, it can live directly on `vim.inspect_pos()`. Don't need a new "inspector" module.

Another reason that a new module doesn't fit here is, we have various modules lie vim.treesitter, vim.diagnostic, that cover topics, and they would have `vim.x.inspect()` functions on them. So it doesn't make sense to invert that relationship and put things in a `vim.inspector` module.